src/language-html/utils/index.js

```diff
@@ -108,10 +108,11 @@ function isScriptLikeTag(node) {
     node.type === "element" &&
     (node.fullName === "script" ||
       node.fullName === "style" ||
+      node.fullName === "mj-style" ||
```

Comment:
I think it should only work if the parser is `mjml`

Reply:
I'll make some change 👍

Reply:
Do you think we should add a "real" `mjml` parser? I'm open to do so.

Reply:
Might be simple fork of current `html` one, it might follow the same behavior at the end, but if it prevent to pollute html parser it could be a good idea

Reply:
#17339

Reply:
Wow thank you so much for this ! I'll rebase on this firstly, and would you prefer to forward options with parser in this func or having a separate version of it and using it only when it matters ? `inferStyleParser` already pass options so to be consistent I guess I should follow it ?

Reply:
Let's wait for a while to see if other maintainers are ok with it. If it's accepted, we'll add a `options.parser === 'mj-style'` check here.

Reply:
#17339 has been merged.

Reply:
I'm updating this PR this week to check the parser 👍